modules/rosa-cluster-autoscaler-ui-after.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+:_content-type: REFERENCE
+[id="rosa-cluster-autoscale-ui-after_{context}"]
+= Enable autoscaling after cluster creation with OpenShift Cluster Manager
+
+You can use OpenShift Cluster Manager to autoscale after cluster creation.
+
+.Procedure
+
+. In OpenShift Cluster Manager, click the name of the cluster you want to autoscale. The Overview page for the cluster has a “Cluster autoscaling” item that indicates if it is enabled or disabled.
+
+. Click the *Machine Pools* tab.
+
+. Click the *Edit cluster autoscaling* button. The *Edit cluster autoscaling* settings window is shown.
+
+. Check the *Enable cluster autoscaling* box at the top of the window. All the settings are now editable.
+
+. Edit any settings you want and then click *Close*.
```

Comment:
@celebdor should we make it ``Close and Save`` (or something along those lines)?  To make sure the user understand this doesn't cancel the update but rather applies it